engine/apps/alerts/migrations/0029_auto_20230728_0802.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.20 on 2023-07-28 08:02
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_management', '0013_alter_organization_acknowledge_remind_timeout'),
+        ('alerts', '0028_drop_alertreceivechannel_restricted_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='alertgroup',
+            name='acknowledged_by_user',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='acknowledged_alert_groups', to='user_management.user'),
+        ),
+        migrations.AlterField(
+            model_name='alertgroup',
+            name='wiped_by',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='wiped_alert_groups', to='user_management.user'),
+        ),
+    ]
```

Comment:
these are here because I set the `related_name` attributes for these two fields [here](https://github.com/grafana/oncall/pull/2653/files#diff-ec133533cae470399923e8147f8935ebc09778b6758bda1dadf4d174dc23861cR254-R343) (`mypy` was complaining that it "couldn't find the related manager" for these, so I strongly typed them [here](https://github.com/grafana/oncall/pull/2653/files#diff-c569330343b5bb0d5520d3c300e93b6d3d0fd07c73509ab3354161eda10e2be4R148-R160)